### DISCUSSION TITLE
Error: /usr/sbin/system_profiler exited with code 1

### DISCUSSION THREAD

**Discussion by AmilaSamith** [rancher-sandbox/rancher-desktop#2255] [Q&A]
Rancher Desktop  release 1.3.0 shows above error when starting it.

<img width="774" alt="Screen Shot 2022-05-22 at 7 06 15 AM" src="https://user-images.githubusercontent.com/80140824/169700058-a30fc77f-50db-4d4b-ab48-3db01a10f14e.png">

OS: MacOS Mojave version 10.14.5

How can I solve this issue?

**Comment by jandubois** [comment]
Rancher Desktop requires macOS Catalina or later; Mojave is unfortunately not supported.

**Reply by nick-manasys** [reply]
Any chance the installer could indicate this in order to avoid disappointment?
